docs/types/value-types.rst

```diff
@@ -39,6 +39,11 @@ Operators:
 * Shift operators: ``<<`` (left shift), ``>>`` (right shift)
 * Arithmetic operators: ``+``, ``-``, unary ``-``, ``*``, ``/``, ``%`` (modulo), ``**`` (exponentiation)
 
+.. warning::
+
+  Integers in Solidity are restricted to a certain range. For example, with ``uint256``, this is ``0`` up to ``2**256 - 1``.
```

Comment:
uint256 is perhaps not the best example, because it covers the full range.

Reply:
Done